balance.go

```diff
@@ -11,6 +11,43 @@ type TransactionStatus string
 // "application_fee_refund", "transfer", "transfer_cancel", "transfer_failure".
 type TransactionType string
 
+// TransactionSourceType consts represent valid balance transaction sources.
+type TransactionSourceType string
+
+const (
+	// TransactionSourceAdjustment is a constant representing a transaction source of dispute
+	TransactionSourceDispute TransactionSourceType = "dispute"
```

Comment:
@brandur The "Type" here actually corresponds to the object type on the expanded "Source", not to the Type on the Transaction

So for a balance transaction related to a Dispute, the transaction type will be "adjustment" but the expanded source object will be a "dispute".